test.md

```diff
@@ -0,0 +1 @@
+"This file should be ignored by git"
```

Comment:
- [ ] Make sure that you added `test.md` and `node_modules/` into `.gitignore` file

Reply:
Where can I find the node_modules?